.github/workflows/workflows-simple-canary-prod-test.yaml

```diff
@@ -0,0 +1,30 @@
+name: workflows-simple-canary-prod-test
+'on':
+  schedule:
+    - cron: "0 * * * *"  # run hourly
+  workflow_dispatch:
+  pull_request: #remove this before merging
+    branches: [ develop ]
```

Comment:
This will be removed before merging. This is here to make it easy to test the GHA.